src/hacbs/components/ImportForm/ImportForm.tsx

```diff
@@ -44,30 +35,27 @@ const ImportForm: React.FunctionComponent<ImportFormProps> = ({ applicationName
     },
   };
 
-  const steps = useImportSteps(applicationName, {
-    onApplicationCreated: setApplicationData,
-    onComponentsCreated: () => setComponentsCreated(true),
-  });
+  const steps = useImportSteps(applicationName);
 
   const handleSubmit = React.useCallback(
-    ({ application, inAppContext }: FormValues) => {
-      const appName = inAppContext ? application : applicationDataRef.current?.metadata?.name;
-      navigate(`/stonesoup/applications/${appName}`);
+    (values: FormValues, formikHelpers: FormikHelpers<FormValues>) => {
```

Comment:
Why keep this separate `ImportForm` component in hacbs folder? Why not just replace the original one with this?

Reply:
Moved contents from hacbs folder to src and removed unused files.